budgetApp1/budgetApp1/TransactionModel.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  TransactionModel.swift
+//  budgetApp1
+//
+//  Created by Lehlohonolo Nkadimeng on 2024/04/14.
+//
+
+import Foundation
+
+// MARK: - TransactionData
+struct TransactionData: Codable {
+    let data: TransactionDataClass
+}
+
+// MARK: - TransactionDataClass
+struct TransactionDataClass: Codable {
+    let transactions: [Transaction]
+    let serverKnowledge: Int
+
+    enum CodingKeys: String, CodingKey {
+        case transactions
+        case serverKnowledge = "server_knowledge"
+    }
+}
+
+// MARK: - Transaction
+struct Transaction: Codable {
+    let id, date: String
```

Comment:
```suggestion
    let transactionID, date: String
```
id is very vague. Please rename to something like suggestion and don't forget to add a coding key